src/components/InngestCommHandler.ts

```diff
@@ -748,8 +753,63 @@ export class InngestCommHandler<H extends Handler, TransformedRes> {
     return this.showLandingPage ?? strBoolean(strEnvVar) ?? true;
   }
 
-  protected validateSignature(): boolean {
-    return true;
+  protected validateSignature(sig: string | undefined, body: Record<string, any>) {
+    if (!this.signingKey) {
+      console.warn(
+        "No signing key provided to validate signature.  Find your dev keys at https://app.inngest.com/test/secrets"
```

Comment:
Unsure about this warning being `/test`, you don't need this in development, so it can be skipped completely. I haven't used signing keys locally and it may be a noisy warning for folks getting started every time a function is called 🤔

Reply:
Fixed by checking for !prod and !signingKey then returning early.  Good point!